srcpkgs/synapse/template

```diff
@@ -12,13 +12,15 @@ depends="python3-jsonschema python3-frozendict python3-canonicaljson
  python3-setuptools python3-bleach python3-Jinja2 python3-psycopg2 python3-lxml
  python3-saml2 python3-treq python3-macaroons python3-sortedcontainers
  python3-typing_extensions python3-ijson"
+checkdepends="$depends python3-parameterized"
 short_desc="Matrix reference homeserver"
-maintainer="Pete <pete@port22.co.uk>"
+maintainer="Joel Beckmeyer <joel@beckmeyer.us>"
 license="Apache-2.0"
 homepage="https://github.com/matrix-org/synapse"
 changelog="https://raw.githubusercontent.com/matrix-org/synapse/develop/CHANGES.md"
 distfiles="https://github.com/matrix-org/synapse/archive/v${version}.tar.gz"
-checksum=5f730958e57b8196ee9352b8abc32da612296f8febf675dec76e0b9d3a322ec5
+checksum=329e5f075fd68835f36a2aa960e6c5fd7e9f82d79797350870f581a2c47f3a42
+make_check=extended
```

Comment:
How long are they taking on CI? Please add a comment justifying it.

Reply:
real  17m13.758s
user  17m5.609s
sys 0m9.753s

So, not really that long. IIRC the `beets` tests take a similar amount of time. I can remove it and have the tests run, I just didn't see any hard and fast guidance about how long tests should be before it's too long.

Reply:
17 minutes is short enough :)

We have 6h for CI runs.

Reply:
Thanks, I removed `make_check=extended`